dotnet/test/common/BiDi/Script/CallFunctionLocalValueTest.cs

```diff
@@ -349,7 +365,7 @@ await context.Script.CallFunctionAsync($$"""
     [Test]
     public void CanCallFunctionWithArgumentSet()
     {
-        var arg = new SetLocalValue([new StringLocalValue("setKey")]);
+        var arg = new SetLocalValue(["setKey"]);
```

Comment:
I propose to not use implicit casting in these tests. I think we can create separate suite especially to verify casting. And your previous RoundTrip suggestion looks good.

Reply:
Like see what magic selenium proposes.

Reply:
Unknown future, still discovering.